dd-java-agent/instrumentation/jax-rs-client/connection-error-handling-jersey/src/main/java/datadog/trace/instrumentation/connection_error/jersey/JerseyClientConnectionErrorInstrumentation.java

```diff
@@ -62,12 +65,14 @@ public static void handleError(
         @Advice.FieldValue("requestContext") final ClientRequest context,
         @Advice.Thrown final Throwable throwable) {
       if (throwable != null) {
-
         final Object prop = context.getProperty(ClientTracingFilter.SPAN_PROPERTY_NAME);
         if (prop instanceof Span) {
           final Span span = (Span) prop;
-          Tags.ERROR.set(span, true);
-          span.log(Collections.singletonMap(Fields.ERROR_OBJECT, throwable));
+          if (!(throwable instanceof RedirectionException)
```

Comment:
Is there a reason to prefer repeating this code rather than having it as a utility function somewhere?

Reply:
Not sure the logic belongs in the `ClientTracingFilter` and introducing a new dependency that would be immediately reverted seems like wasted effort.  Ok with this landing and then adding an error logger to the decorators during 0.28.0 adoption?